.travis.yml

```diff
@@ -20,24 +20,6 @@ env:
 
 matrix:
   include:
-    # This environment tests that scikit-learn can be built against
-    # versions of numpy, scipy with ATLAS that comes with Ubuntu Trusty 14.04
-    # i.e. numpy 1.8.2 and scipy 0.13.3
-    - env: DISTRIB="ubuntu" PYTHON_VERSION="2.7" CYTHON_VERSION="0.23.5"
-           COVERAGE=true
-      if: type != cron
-      addons:
-        apt:
-          packages:
-            # these only required by the DISTRIB="ubuntu" builds:
-            - python-scipy
-            - libatlas3-base
-            - libatlas-dev
-    # Python 3.4 build
```

Comment:
We still need a run with minimum requirements, so make sure that we have a python 3.5 run and that minimum / alternative variants of dependencies are still tested

Reply:
I think we still want to support Ubuntu LTS package distributions. Can you check whether trusty providers python 3.5+ with those same numpy/scipy releases?